x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/integration_tests/response_actions_log.test.tsx

```diff
@@ -1490,6 +1490,14 @@ describe('Response actions history', () => {
   });
 
   describe('Actions filter', () => {
+    const featureFlags = {
```

Comment:
please don't use a mutable object between tests. Can you convert this to a function? or re/set it in the `beforeEach()`

Reply:
thanks for indicating this for me... _again_... 🤦
i already had a keep note, but it wasn't enough... so as a next step, now i put a sticky note on my monitor to remember this 😅 

9267fb6f8d9f9813515565a05aa8b1efd3be004b